### DISCUSSION TITLE
Buffer size?

### DISCUSSION THREAD

**Discussion by convinced-prog** [publicsamples/Oi-Grandad#52] [Q&A]
I see Oi Grandad is at his old tricks again.
I had it running  both in a vst host and in a DAW but now he's started to insist on a multiple of 8 buffer which for some reason my vst host can't do or doesn't offer the choice.
While I'm here. Can you make 'Q' ui scalable as it refuses to fit in my vst host window. Too bg and clips certain controls.
Sorry to be a nuisance
Thanks

**Comment by RickTaylor79** [comment]
Hey, sorry for the late reply. Never could multi-task. 

I'm no Windows expert, but this buffer thing sounds like a configuration problem on your end. Really there is no reason to be using a buffer that's not a multiple of 8.. there isn't a host out there that does not support this. Hopefully you've got it figured out, but if not I reckon a driver misconfiguration could be the culprit. Check any 3rd party audio drivers, try using the default direct sound..

Q has a scaling option in Settings & Presets, unfortunately however, I'm lazy so it's a nondescript grey box labeled "UI Zoom Factor". 

Also, you asked awhile ago about standalone..

https://github.com/publicsamples/Oi-Grandad/releases/tag/1.1.8

**Comment by convinced-prog** [comment]
Hey thanks for that I was running Grandad in a VST host which he was
reasonably 'happy with' before losing his pension book.
Yes I had a monumental battle with my audio and midi drivers which after
loading excellent third party programs seems to have resolved things
somewhat.
Windows midi support is nothing short of pathetic. Nearly drove me back to
Linux.
As for his pal 'Q', does this 0.86 mean I can sleep at night?
Too scared to turn my PC off in case it stops working!!
Just a one stop installation. Do I need a new vst file?
Anyway I'm on the brink of going for the 'expanded' version, whatever that
is!
Good job so far
Will keep you informed.
Many thanks All the best

**Comment by RickTaylor79** [comment]
Good to hear you got it figured out. Yeah, it's insane how bad the default MIDI and audio drivers are in windows. Mac and Linux are superior in this regard, it's been like this forever. 

Q is a lot more stable, really, I kid you not. Also a lot more CPU friendly.. Yeah, just uninstall any old files (old presets will break things) and install the vst. That's it. No additional content to install unless you want it (this is where the expansion comes in, extra sounds, presets etc).  Your feedback has been super helpful. [Email me](mailto:modularsamples@gmail.com), and I'll sort that expansion out.

**Comment by convinced-prog** [comment]
Thanks again. I'm really determined to make this thing work, in fact I've
been so obsessed with it I've let other things slip.
Any help will be greatly appreciated.
Sleep well!
